warehouse/cli/search/reindex.py

```diff
@@ -57,6 +57,8 @@ def reindex(config, **kwargs):
     """
     client = config.registry["elasticsearch.client"]
     db = Session(bind=config.registry["sqlalchemy.engine"])
+    number_of_replicas = config.registry.get("elasticsearch.replicas", 0)
+    refresh_interval = config.registry.get("elasticsearch.interval", "5s")
```

Comment:
Is there any reason you've chosen ``5s`` here? I'm not against that number just curious if there's some significance to it or not.